### PR TITLE
Fix Facebook auto sign in test by properly logging out and modifying the log out cookie

### PR DESCRIPTION
**Pull request by nlindblad**
If the log out is performed via the menu and the sign out cookie (`GU_SO`) is updated to a time 24 hours ago, the actions fall within the scope of the auto sign-in feature.

### CODE REVIEW

File: common-test-lib/src/main/scala/com/gu/integration/test/util/CookieUtil.scala [hunk 1]
```diff
@@ -11,6 +11,11 @@ object CookieUtil {
     driver.manage().deleteCookieNamed(cookieName)
   }
 
+  def createCookie(cookieName: String, content: String)(implicit driver: WebDriver) = {
+    val cookie = new Cookie(cookieName, content)
+    driver.manage().addCookie(cookie)
+  }
+
   /**
```
Comment (jamesoram) [inline common-test-lib/src/main/scala/com/gu/integration/test/util/CookieUtil.scala]: this looks like something that could be useful to add to the framework, please create a PR here: https://github.com/guardian/scala-automation

**Comment by jamesoram** [comment]
this card has the business rules https://trello.com/c/L4VYtq6I/67-auto-sign-in-of-facebook-users-not-working-on-next-gen

**Comment by jamesoram** [comment]
looks good, but I think the cookie helper belongs in the framework

File: common-test-lib/src/main/scala/com/gu/integration/test/util/CookieUtil.scala (hunk 1 above)
Comment (nlindblad) [inline common-test-lib/src/main/scala/com/gu/integration/test/util/CookieUtil.scala]: Marked them both as deprecated and now using `CookieManager` from scala-automation.

**Comment by nlindblad** [comment]
Bumped up the scala-automation version and after adding the local storage value clearing, auto sign-in works as expected.

**Comment by jamesoram** [comment]
:+1:

**Pull request merged**